### DISCUSSION TITLE
New Feature Request for Stella - Random Game Generator

### DISCUSSION THREAD

**Discussion by livingroomarcade** [stella-emu/stella#991] [Ideas]
Hello coders,
I'm just a Stella enthusiast.  (I play Atari.)  I'm not a coder.  
Can I make a new feature request?  
In a future version of Stella, could there possibly be a prominent button somewhere to click to let Stella choose a **random game** for you to play from among those available in your ROMS folder?
Would be a great way to explore some of those many great Atari games that you've never tried before.

Thanks for making Stella.  It's a great Atari 2600 emulator.  I love using it!

**Comment by thrust26** [comment]
Done with Stella 7.0.

**Reply by livingroomarcade** [reply]
And then some.  It's awesome!  Thanks, thrust26.
